### PR TITLE
adjustments for release

### PR DESCRIPTION
**Pull request by bernardoaraujor**
# Description of change

Updated docs to reflect the latest release of Access

# Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)

# Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my changes

### CODE REVIEW

File: iota-access/1.0/tutorials/get-started.md [hunk 1]
```diff
@@ -193,32 +207,28 @@ To run the Access Mobile Client, you need [Android Studio](https://developer.and
...
     Password: IOTApass1234
     ```
 
-7. Add the IP address of your Access server (Raspberry Pi) to the **Settings** page
+5. Add the IP address of your Access Server (Raspberry Pi) to the **Settings** page
 
     ![Settings](../images/access-server-settings.gif)
 
-8. Go to **Delegation editor** and create a new policy by selecting User 1 and Action 1
+6. Add the IP address of your Access Policy Store (Linux server) to the **Settings** page
```
Comment (JakeSCahill) [inline iota-access/1.0/tutorials/get-started.md:225]: Can we merge this with step 5?

So ` Add the IP address of your Access server and policy store to the **Settings** page`

File: iota-access/1.0/tutorials/get-started.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Future version of Access will allow you to connect to Hornet node software so th
             "host": string,
             "port": number
         },
-        "iri": {
+        "node": {
```
Comment (JakeSCahill) [inline iota-access/1.0/tutorials/get-started.md:77]: We mention the `iri` field in one of the steps above, can we make sure to change that to `node`?

Also, if users no longer need a local IRI node, we can remove it from the prerequisites.

File: iota-access/1.0/tutorials/get-started.md (hunk 1 above)
Comment (JakeSCahill) [inline iota-access/1.0/tutorials/get-started.md:225]: Can we merge this step with step 5?
You add both on the same page so no need to separate them anymore

Comment (bernardoaraujor) [reply iota-access/1.0/tutorials/get-started.md:225]: sure! `addresses` as plural

File: iota-access/1.0/tutorials/get-started.md (hunk 2 above)
Comment (bernardoaraujor) [reply iota-access/1.0/tutorials/get-started.md:77]: we still haven't sorted out that issue, so it's fine to leave it like that for the moment.

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**